src/02-data-types.js

```diff
@@ -22,7 +24,10 @@ function getLengthOfString(str) {}
  *  convertToNumber("111");
  *  //> 111
  */
-function convertToNumber(val) {}
+function convertToNumber(val) {
+  newVal = Number(val);
```

Comment:
Here we could have returned `Number(val)` - no need for a variable we're no using it again in our code!